src/state/slices/opportunitiesSlice/selectors/lpSelectors.ts

```diff
@@ -119,7 +130,7 @@ export const selectAggregatedEarnUserLpOpportunity = createDeepEqualOutputSelect
       underlyingToken1AmountCryptoBaseUnit,
       cryptoAmountPrecision: aggregatedLpAssetBalance,
       // TODO(gomes): use base unit as source of truth, conversions back and forth are unsafe
-      cryptoAmountBaseUnit: toBaseUnit(aggregatedLpAssetBalance, assets[lpId]?.precision ?? 0),
+      cryptoAmountBaseUnit: toBaseUnit(aggregatedLpAssetBalance, lpAsset?.precision ?? 0),
```

Comment:
similarly here

Reply:
Set to the default 0.5% UNI-V2 settings in 94a54a6896955feef3c15bcf667881cb21d27fba

Reply:
43b517b9d